src/components/DateRangePicker.js

```diff
@@ -547,7 +582,7 @@ const DateRangePicker = React.createClass({
         fontSize: StyleConstants.FontSizes.MEDIUM,
         paddingLeft: isLargeOrMediumWindowSize ? 10 : 0,
         paddingTop: 10,
-        maxWidth: 250,
+        maxWidth: window.innerWidth > 450 && 250,
```

Comment:
I'm not seeing any css errors, the style just gets applied when the condition is met and doesn't get applied when the condition isn't met. Maybe better though is to just default it to the inherited property from div if it isn't applied? I'll do that.
with condition met:
![screen shot 2017-02-16 at 6 10 04 pm](https://cloud.githubusercontent.com/assets/1081167/23048460/37f49b1a-f473-11e6-8484-93254483bb1d.png)
with condition not met:
![screen shot 2017-02-16 at 6 09 41 pm](https://cloud.githubusercontent.com/assets/1081167/23048461/37fa09ec-f473-11e6-962d-942be96afbd0.png)